File: katas/evenOrOdd.js
// Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.

function even_or_odd(number) {
    if(number % 2 > 0 || number % 2 < 0){
        return "Odd"
    }
    return "Even"
}

console.log(even_or_odd(45));

File: Rust/evenOrOdd.rs
fn main() {
    let test = even_or_odd(1);
    println!("{}", test);
}

fn even_or_odd(i: i32) -> &'static str {
    if i % 2 == 0 {
        "Even"
    }else {
        "Odd"
    }
}


// fn even_or_odd(i: i32) -> &'static str {
//     match i % 2 {
//       0 => "Even",
//       _ => "Odd",
//     }
// }


// ternaire
// fn even_or_odd(i: i32) -> &'static str {
//     if i % 2 == 0  { "Even" } else { "Odd" }
//   }